src/components/ratings/CertificatesList.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { StyleSheet, View } from 'react-native'
+import CertificateLabel from '@components/ratings/CertificateLabel'
+
+const CertificatesList = () => {
```

Comment:
I don't know about the certificates but you might be able to store each one in an array and use the js map function to create a certificate label for each certificate in the array. This way the component is more generalised and it is easier to add more certificates or remove certificates by adding or removing from the array.